source/support.tex

```diff
@@ -27,6 +27,7 @@
 \ref{support.limits}      & Implementation properties &
   \tcode{<cfloat>}, \tcode{<climits>}, \tcode{<limits>}, \tcode{<version>}    \\ \rowsep
 \ref{cstdint}             & Integer types             &   \tcode{<cstdint>}   \\ \rowsep
+\ref{stdfloat.syn}        & Header \libheader{stdfloat} synopsis & \tcode{<stdfloat>} \\ \rowsep
```

Comment:
I don't think there is a pre-existing example of a "Header <xxx> synopsis" being at top-level just under a main library clause (and thus appear in this table), though that's what the paper asked... (which also makes this table looks particularly weird)
Maybe we should still do a [stdfloat] "Floating-point types" with [stdfloat.syn] as subclause (and maybe a one-line general like [cstdint])? Or should we open a new convention here so that in future synopsis can directly appear under a clause?

Reply:
> I don't think there is a pre-existing example of a "Header <xxx> synopsis" being at second-level just under a main library clause (and thus appear in this table)

You're right.

Reply:
I agree. We should either start a new subsection "Floating-point types" with additional wording, perhaps relating to C, or otherwise perhaps we can rename "Integer types" to "Numeric types" (or "Arithmetic types") and have both <cstdint> and <stdfloat> in it. I'll open an issue,

Reply:
After shuffling words around in an attempt to make things more clear (with most of the words ending up in [basic.extended.fp]), it ended up with the header `<stdfloat>` being defined entirely by its synopsis.  There were no English words left in the `<stdfloat>` section.  I found this weird, but I didn't know what to do about it.  LWG thought it was okay and didn't ask me to change it.

I have no objection to creating an almost empty section above [stdfloat.syn] so that `<stdfloat>` is organized more like all the other headers.

Reply:
Yes, it's not entirely clear, but we'll take a look after the merging. I'm curious in particular what C is going to do and how this would look after we rebase on C23.